executor-scripts/linux/wireguard-quick

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+[ -n "$VERBOSE" ] && set -x
+[ -z "$IF_WIREGUARD_CONFIG_PATH" ] && IF_WIREGUARD_CONFIG_PATH="/etc/wireguard/$IFACE.conf"
```

Comment:
I guess this should go away as it's not used

Reply:
Oh yes!, the $VERBOSE part is right?